feat: RTL support

vltansky opened the pull request:
pretty small fix for RTL support by just enforcing it to be LTR as numbers anyway always written left to right.

guilhermerodz commented:
hi @vitansky i would love to add support to RTL, still I don't know what's a good example for this. this PR does not achieve the expected behavior as the CSS attribute is not enough to affect input's behavior.

vltansky commented:
Hi, I've added an example to switch to RTL/LTR in the demo website. 
Also, I've added the style in the right place, my bad :)

before:
https://share.cleanshot.com/9mpvmwsr

after:
https://share.cleanshot.com/yX8bjvt8

it should behave just the same as LTR.

guilhermerodz commented:
hi @vltansky I'm still not sure how to achieve the behavior you mentioned.

can you provide me with a preview/link?
